<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Dialog</class>
 <widget class="QDialog" name="Edit_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size:13px;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Food name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="fdname_plainTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="recipe_label">
       <property name="text">
        <string>Recipe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="recipe_plainTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>171</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ingred_label">
       <property name="text">
        <string>Ingredients                                      Please write in the following format without brackets - [name: quantity unit] </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="ingred_plainTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>161</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="save_pushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>418</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
